bpf/lib/nodeport.h

```diff
@@ -152,8 +152,11 @@ static __always_inline bool nodeport_uses_dsr6(const struct ipv6_ct_tuple *tuple
 	return nodeport_uses_dsr(tuple->nexthdr);
 }
 
-static __always_inline bool nodeport_nat_ipv6_needed(struct __ctx_buff *ctx,
-						     union v6addr *addr)
+/* TODO(brb): once we implement MASQ for IPv6, we can move snat_v{4,6}_needed()
+ * to lib/nat.h, as then the helper function won't depend the dsr checks.
+ */
```

Comment:
nit: Could we track this on GH as TODO instead of comment here?

Reply:
Updated https://github.com/cilium/cilium/issues/11018.